README.md

```diff
@@ -64,18 +64,19 @@ Finally take a glance at the API/documentation workflow below. This should be do
 5. Type `ls -als` (if you have the alias, this is `ll`) compare the directory structures in this directory tree to that in the original repo to see what is different (ignore files at this point).  Nothing to do here, just get familiar with the differences.
 6. Move the `.git` directory from the main directory to the cookiecutter directory. From the main directory type `mv ../.git .`.
 7. Create a new branch for all the changes, e.g., `git checkout -b cookierelease`.
-8. Copy the code from the old repo to the cookiecutter repo, without overwriting files in the destination (i.e., use `cp -n` or `cp --no-clobber`.  e.g., from the cookiecutter directory where you currently are, type `cp -n -r ../src .` if the old code is already in a directory `src`.  If there is no src directory, it will be something like `cp -n -r ../diffpy ./src`.
-9. From the cookiecutter directory type `git status`.  You will see a list of files that have been modified, deleted or are untracked.  Untracked files are in the cookiecutter but not in the original repo, deleted files are in the original but haven't been moved over, and modified files are in both but have been changed.
-10. Let's now copy over any documentation, similar to what we did with the src files.  We want to copy over everything in the `doc/<path>/source` file from the old repo to the `doc/source` file in the new repo.  In some old pacakges, `<path>` will be something like `manual` and in others it will just not exist.
+8. Look into `.github/workflows`, change `c_extension` and `headless` parameters to `true` if the package has C extensions or is a gui package.
```

Comment:
lead this with the target situtuation, so somethin glike "If the package has C extensions....." which will mean I only have to read the first few words and not the whole thing if I don't have c....